src/helpers/verifyAddUserForm.js

```diff
@@ -0,0 +1,29 @@
+import { displayErrorMessagesAddUSer } from '../helpers/displayErrorMessagesAddUser'
+import { errors } from '../helpers/errorMessages'
+import { displayErrorFor } from './errorMessageHandle'
+import users from '../data/users.json'
+
+export const verifyAddUserForm = (dispatchGetFormData, dispatchGetFormError, dispatchGetDoubleUsers, formData) => {
+
+  const allFieldsFilledIn = formData.name.length > 0 &&
+    formData.lastName.length > 0 &&
+    formData.email.length > 0 &&
+    formData.phone.length > 0
+
+  const emailValidationRegexp = /^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/gi
+  const emailValidation = formData.email.match(emailValidationRegexp)
+
+  const phoneVelidationRegexp = /[0-9]+$/gi
```

Comment:
Ha, dość naiwne założenia co do numeru ;) Jeżeli już robisz walidację numeru telefonu via RegExp, rób to dobrze :) Co ze spacjami (`123 34 54 65`)? Albo numerem kierunkowym (`(58) 345 134 234`)? Albo myślnikami? Albo kierunkowym kraju? Twój RegExp wprowadza ogromne ograniczenia.